docs/source/installation.rst

```diff
@@ -27,6 +27,47 @@ Android on Ubuntu 18.04 (64bit)
     # add the following line at the end of your ~/.bashrc file
     export PATH=$PATH:~/.local/bin/
 
+Android on Windows 10
+~~~~~~~~~~~~~~~~~~~~~
+
+To use buildozer in Windows 10 you need first to enable Windows Subsystem for Linux (WSL) and install a Linux distribution: https://docs.microsoft.com/en-us/windows/wsl/install-win10.
+
+These instructions were tested with WSL 1 and Ubuntu 18.04 LTS. 
+
+With WSL and Ubuntu installed on your Windows 10 machine, open Ubuntu and run these commands:
+
+::
+
+    sudo apt update
+    sudo apt install -y git zip unzip openjdk-8-jdk python3-pip autoconf libtool pkg-config zlib1g-dev libncurses5-dev libncursesw5-dev libtinfo5 cmake libffi-dev libssl-dev
+    # Use here the python version you need
+    sudo apt install -y python3.7-venv
+    # Create a folder for buildozer
+    mkdir /mnt/c/buildozer
+    cd /mnt/c/buildozer
+    python3.7 -m venv venv-buildozer
+    source venv/bin/activate
+    python -m pip install --upgrade pip
+    python -m pip install --upgrade wheel
+    python -m pip install --upgrade cython 
+    python -m pip install --upgrade virtualenv 
+    python -m pip install --upgrade buildozer
+    # Add the following line at the end of your ~/.bashrc file
+    export PATH=$PATH:~/.local/bin/
+    # Restart your WSL terminal to enable the path change
```

Comment:
I feel like most of this could reuse what already holds true in the Ubuntu install section above, that way we don't repeat ourselves

Reply:
Yes

Reply:
I 'm gonna change it